packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -223,6 +223,14 @@ class NotificationDialog extends React.Component {
       message,
       ...customProps
     } = this.props;
+    if (acceptAction === undefined && primaryAction === undefined) {
+      // eslint-disable-next-line no-console
+      console.warn('acceptAction or primaryAction prop must be given.');
+    }
+    if (rejectAction === undefined && secondaryAction === undefined) {
+      // eslint-disable-next-line no-console
+      console.warn('rejectAction or secondaryAction prop must be given.');
+    }
```

Comment:
I think we only need a warning if none of `acceptAction`, `rejectAction`, `primaryAction` or `secondaryAction` is defined.  

As long as one of those four is defined we can exit the notification-dialog which was the main reason `primaryAction` and `acceptAction` were marked as required in previous versions.

Reply:
Resolved in 2957b32